@xen-orchestra/web-core/lib/components/input/InputLabel.vue

```diff
@@ -0,0 +1,74 @@
+<!-- v2.0 -->
+<template>
+  <div :class="status" class="input-label">
+    <UiIcon :icon class="left-icon" />
+    <span :class="{ required }" class="typo c2-semi-bold label"><slot /></span>
+    <a v-if="href" :href class="link">
+      <span class="typo p3-regular-underline">{{ $t('learn-more') }}</span>
```

Comment:
From what I've seen, the translation key is only present in XO Lite and should be moved to `web-core`.